error_test.rb

```diff
@@ -0,0 +1,91 @@
+require_relative 'environment.rb'
+
+class ErrorTest
+  include Environment
+  attr_accessor :board, :depot, :control, :dictionary, :poker
+
+  def initialize
+    @board = { players: Hash.new, control: Hash.new, status: nil }
+    @control = Environment::Control.new
+    @depot = Environment::Depot.new
+    @dictionary = Environment::Dictionary.new
+  end
+
+  def new_worker name: nil
+    begin    
+      raise StandardError, "Is not posible to have a null name" if name.nil?
+      raise StandardError, "Is not posible to have an empty name"if name.empty?
+      worker = Environment::Human::Worker.new(name: name)
+      @board[:players][name] = worker
+    rescue StandardError => e
+        puts "Error: #{e.message}"
+    end
+  end
+
+  def new_mission name: nil, objective: nil, pack: nil
+    begin
+      error_message = ""
+      if name.nil?
```

Comment:
Check this approach 
```
error_message = "Name is null. " if name.nil?
error_message += "Objective is null. " if objective.nil?
```
`raise StandardError, error_message unless error_message.empty?`

Single line statements